src/components/EditableField/__tests__/EditableField.test.tsx

```diff
@@ -1161,35 +1162,7 @@ describe('enter press', () => {
     })
   })
 
-  test('Pressing Enter: should escape on validation rejection', () => {
-    const wrapper = mount(
-      <EditableField
-        name="company"
-        value={{ value: '1234567', id: '1' }}
-        validate={() => Promise.reject(new Error('Oops'))}
-      />
-    )
-
-    // @ts-ignore
-    const escapeSpy = jest.spyOn(wrapper.instance(), 'handleFieldEscapePress')
-    const stateSpy = jest.spyOn(wrapper.instance(), 'setState')
-    wrapper.instance().forceUpdate()
-
-    const input = wrapper.find('input').first()
-    // @ts-ignore
-    input.getDOMNode().value = '1234567'
-    input.simulate('keydown', { key: 'Enter' })
-
-    const f = flushPromises()
-    jest.runAllImmediates()
-
-    f.then(() => {
-      expect(escapeSpy).toHaveBeenCalled()
-      expect(stateSpy).toHaveBeenCalledWith({ disabledItem: [] })
-    })
-  })
```

Comment:
@tinkertrain None of these tests were finishing before because we weren't calling `done()` in the callback. I fixed all of the tests up, but can't figure out what we should do with this test here so I've removed it for now. Any idea?

Reply:
Hi! I'm having problems understanding what needs to happen here, running the test as is seems to be running the assertions but I think I'm wrong on some assumption here, considering this and the next tests are rejected...

Reply:
@tinkertrain The assertions actually aren't running synchronously, so they do fail, but don't break the test because they fail after it finishes. If we add the `done` argument to the test and invoke it in the `f.then` block, then the assertions will run within the test and fail. I cannot for the life of me, get that spy to be called. However, I can verify that it is reached.

Reply:
Oh I see, I did try several combinations too, within the `catch` or the second argument on `then` but couldn't make it work either, it's fine by me to delete or comment them out :)

Reply:
Maybe leave at least one and comment out and leave a comment of why they're commented? might be usefult at some point in another instance that is similar

Reply:
Sounds good. I think I'll just leave them deleted then for the same of not having commented out stuff here. We can always refer back to this PR. There is coverage around escaping and the test coverage is not impacted by this removal.